framework/helpers/BaseUrl.php

```diff
@@ -86,8 +86,8 @@ class BaseUrl
      * @param bool|string $scheme the URI scheme to use in the generated URL:
      *
      * - `false` (default): generating a relative URL.
-     * - `true`: returning an absolute base URL whose scheme is the same as that in [[\yii\web\UrlManager::hostInfo]].
-     * - string: generating an absolute URL with the specified scheme (either `http` or `https`).
+     * - `true`: returning an absolute base URL whose scheme is the same as that in [[\yii\web\UrlManager::$hostInfo]].
+     * - string: generating an absolute URL with the specified scheme (either `http`, `https` or empty string).
```

Comment:
maybe add a hint `or empty string, for protocol relative URL`? it is not directly obvious why one would pass an empty string here, also searching for "yii protocol relative url" would find this text better than now.